### PR TITLE
Add namespace attributes for images and videos

### LINKED ISSUE #16: Google Search Console identifies sitemap issue
Upon adding my generated sitemap to Google Search Console, I'm met with this message:

Screen Shot 2020-12-31 at 2 22 21 PM

I don't really understand and am having a hard time googling the error, but I did find the same issue being resolved in another sitemapping repo here: https://github.com/omz13/kirby3-xmlsitemap/issues/12 

Here's the sitemap that was generated and uploaded using the `:aws` storage option: https://luckycasts-sitemap.s3.amazonaws.com/sitemap.xml

This also may not really matter, since Google says it's readable. I'm going to wait a day or two and see if it actually impacts crawlers.

### PR DESCRIPTION
**Pull request by stephendolan**
Closes #16 

I kept running into errors like this when checking my sitemap that's chock full of videos:

> Namespace prefix video on thumbnail_loc is not defined

I checked Google's docs, and for both Images and Videos they say that we need to include these namespaces:
- https://developers.google.com/search/docs/advanced/sitemaps/video-sitemaps#xml-namespace
- https://developers.google.com/search/docs/advanced/sitemaps/image-sitemaps#xml-namespace

I'm sure there's something "smarter" that could be done to only add these when videos/images are present, but I don't think that the extra characters will be of much consequence.

I also wasn't sure if Bing has their own namespace or something, but I couldn't find anything on their documentation so I'm sort of assuming that the whole internet is letting Google define the namespaces.

I'm running this branch on production for LuckyCasts (https://luckycasts.com/sitemap.xml), and have verified that the Google Search Console errors are gone and browsers can parse the file too.

### CODE REVIEW

**Comment by jwoertink** [comment]
Interesting... I guess I must have missed the errors from my sites that include videos. Sounds like this is a good change, but my only question is, does this affect sites that don't include images and videos? I actually forget how this shard works since I rarely need to change it 😅  but I think we know when video or images are included https://github.com/jwoertink/sitemapper/blob/master/src/sitemapper/builder.cr#L61 maybe we can use that to say "if video/image included, enable the extra namespace"?

**Comment by stephendolan** [comment]
@jwoertink I can confirm that this works without video/image elements present. I removed that from the generation logic for LuckyCasts, and validators still accept it. For some reason I can't attach XML files to GitHub comments, but I can send it in Discord if you'd like to verify yourself!

I don't think that having a bunch of namespaces is out of the ordinary for XML and would probably advocate for keeping it simple in this instance, but if you'd rather I update this with some refactors then I'm happy to do so!

The trick is that the namespace definition happens before we're iterating through paginator items, as well as that there isn't a super clean way to dynamically include hash elements. I could probably figure something decent out, though!

**Comment by jwoertink** [comment]
Ah. ok, in that case then it's good enough for me 👍  Thanks!

**Review by jwoertink — APPROVED** [review]

**Comment by stephendolan** [comment]
Nice, sounds good to me, too! Whenever you merge this, @jwoertink, if you wouldn't mind cutting a release that would be awesome 🥰 

By the way, if you ever want help with managing this repo I'd totally be down to be a maintainer!

**Comment by jwoertink** [comment]
I wonder why the jobs aren't running 🤔

**Comment by stephendolan** [comment]
That's really odd... I tried updating the workflow in case the "branch: [*]" was messing with it to one that I'm positive work with my Decorator shard, but it's still not doing anything.

**Comment by stephendolan** [comment]
Oh, @jwoertink I think they disabled your repo from using actions since there hasn't been activity in 60 days. Here's a message from the Actions tab:

>  This scheduled workflow is disabled because there hasn't been activity in this repository for at least 60 days.

I think you can go in to the settings and toggle Actions off/on to get it re-enabled?

**Comment by stephendolan** [comment]
🎉

**Comment by stephendolan** [comment]
Okay, I looked into conditionally adding this a bit anyways and it was getting into a pretty large refactor. Tests are passing, and this will at least ensure no invalid site maps are generated!

**Comment by jwoertink** [comment]
Ok. We can always look at adding that later as a refactor. This is fine for now.

**Pull request merged**
